spec/dummy/app/views/welcome/admin_example.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render 'govuk_publishing_components/components/title', title: "Hello! I am an example admin page" %>
+<h1 class="govuk-heading-xl">Hello! I am an example admin page</h1>
```

Comment:
Why this change? Isn't this is what the title component is for? If you need to add a class and the component doesn't allow, we should probably extend it rather than not use it. Or was it rendering a h2 or something previously?

Reply:
Since we're trying to use the GOVUK Frontend components (which, for example, only have margins at the bottom) and layout for the admin interface I switched to using a simple heading instead of a title component

Reply:
Ok, I think that makes sense. I'm used to using the existing gem components everywhere so maybe need to get out of that mindset.

Reply:
I agree, I intend to do the same (use gem components everywhere possible), but in this case the title component brings more than I need to the example.

Reply:
Perhaps the 'core' (https://github.com/alphagov/govuk-frontend/tree/master/src/core) parts of GOV.UK Frontend do not need to be wrapped but any 'components' should.

One of the best bits of having everything wrapped as components is if we need to change any markup (for example the class names change) it can be done in one place.